### ISSUE TITLE
Allow customization of email template when replying to issues

### ISSUE THREAD

**Issue by daviddalpiaz** [PrairieLearn/PrairieLearn#3029]
Currently all email responses created for responding to issues appear in this format:

```
Subject: Reported PrairieLearn Issue

Hello <full-student-name>,
Regarding the issue of:

"<Their reported issue here>"

We've...
```

Could we create the ability to customize this template? For example, add course and question and maybe user information in the subject? Use only student first name in greeting? Etc.

**Comment by echuber2** [comment]
A faculty member raised the issue of TAs not being in the loop if an instructor directly emails a reply to a student issue. It might be convenient to have a checkbox to automatically CC the current course staff. E.g., any staff member who has student data access for the current term could be CC'd.

**Comment by jonatanschroeder** [comment]
A better solution might be to implement direct in-PL communication, see #4093.

**Comment by nwalters512** [comment]
Yeah, that's what we'll do in the long run. I'm going to close this issue in favor of that.
